### DISCUSSION TITLE
LFS Git не загружает большие файлы

### DISCUSSION THREAD

**Discussion by pussypistolss** [torin-asakura/dream-team#104] [Q&A] [answered]
Мне нужно запушить файлы 100мб, но гит сначала начинает загружать файлы, а потом выдает эту ошибку: 
![image](https://github.com/torin-asakura/dream-team/assets/147274591/e09273e6-f052-445e-a820-323e610ea32b)
Я уже создала пулл реквест и все прошлые файлы, которые мне нужно было загрузить запушились, скопировав репозиторий с помощью ssh. А LFS почему-то работает только через ssh. Но в этот раз почему-то файлы не запушились. Все эти команды я прописывала:
![image](https://github.com/torin-asakura/dream-team/assets/147274591/e95317aa-09b8-4a98-bcb9-a1a974e84c7e)

**Comment by Nelfimov** [comment]
Можешь дать логи что пишет на второй команде? 

`git lfs track ...`?

**Comment by Nelfimov** [comment]
ну или вот так

https://stackoverflow.com/questions/51724041/git-lfs-error-failed-to-push-some-refs-to

**Reply by pussypistolss** [reply]
`orlov@DESKTOP-P3BE6JI MINGW64 /e/JOB/ssh repa/dream-team (feat/add-3d-sphere)`
`$ git config lfs.contenttype 0`

`orlov@DESKTOP-P3BE6JI MINGW64 /e/JOB/ssh repa/dream-team (feat/add-3d-sphere)`
`$ git push`
`Uploading LFS objects: 100% (2/2), 449 MB | 0 B/s, done.`
`Enumerating objects: 13, done.`
`Counting objects: 100% (13/13), done.`
`Delta compression using up to 4 threads`
`Compressing objects: 100% (10/10), done.`
`Writing objects: 100% (10/10), 245.65 MiB | 2.83 MiB/s, done.`
`Total 10 (delta 4), reused 0 (delta 0), pack-reused 0`
`remote: Resolving deltas: 100% (4/4), completed with 2 local objects.`
`remote: error: Trace: 3e4f77c1801c3bfc6484e79b965f80dd1529c0df8322d25024d756f56e6994a1`
`remote: error: See https://gh.io/lfs for more information.`
`remote: error: File 3D/CompressedSphere.glb is 279.67 MB; this exceeds GitHub's file size limit of 100.00 MB`
`remote: error: GH001: Large files detected. You may want to try Git Large File Storage - https://git-lfs.github.com.`
`To github.com:torin-asakura/dream-team.git`
` ! [remote rejected] feat/add-3d-sphere -> feat/add-3d-sphere (pre-receive hook declined)`
`error: failed to push some refs to 'github.com:torin-asakura/dream-team.git'`
Первый совет по ссылке не помогает. Сейчас второй попробую.

**Comment by pussypistolss** [comment]
> Можешь дать логи что пишет на второй команде?
> 
> `git lfs track ...`?

`$ git lfs track "*.glb"`

`"*.glb" already supported`
В файлике .gitattributes все нужные мне форматы есть

![image](https://github.com/torin-asakura/dream-team/assets/147274591/b4c595d6-2ec9-4262-b7f9-9285794e6af3)

**Comment by Amiditin** [comment, marked as answer]
@pussypistolss у меня на github файлы начали отправляться только после этой команды:
```
git lfs migrate import --include="*.glb"
```
И попробуй запушить только .glb файл.

**Reply by pussypistolss** [reply]
Спасибо большое!
Все работает
